static/templates/communication-costs.hbs

```diff
@@ -0,0 +1,85 @@
+<div class="panel__row">
+  <h4 class="panel__title">Disbursement information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Amount</td>
+      <td class="panel__data">{{ currency disbursement_amount }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Report year</td>
+      <td class="panel__data">{{ report_year }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Transaction date</td>
+      <td class="panel__data">{{ datetime transaction_date pretty=true }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Communication class</td>
+      <td class="panel__data">{{ communication_class }}</td>
+    </tr>
+    <tr>
+      <td class="panel__term">Communication type</td>
```

Comment:
If we display this we would want to decode it as well:
 DM - direct mail
 O - other
 TM - telegram!
 TP - telephone